src/components/country/Country.jsx

```diff
@@ -0,0 +1,68 @@
+import './country.css';
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { BsFillArrowRightCircleFill } from 'react-icons/bs';
+import {
+  searchByRegion,
+  showAllCountries,
+} from '../../redux/countries/countriesAction';
+
+const Country = () => {
+  const {
+    countriesData, loading, success, error, region, searchTerm,
+  } = useSelector((state) => state.country);
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(showAllCountries());
+
+    if (region) {
+      dispatch(searchByRegion(region));
+    }
+
+    if (error) {
+      dispatch(error);
+    }
+  }, [dispatch, error, success, region]);
+
+  const data = countriesData.filter((item) => item.name.common.toLowerCase().includes(searchTerm));
+
+  return (
+    <section className="country-container">
+      {loading ? (
+        <h1>Loading</h1>
+      ) : (
+        data.length > 0
+        && data.map((item, index) => (
+          <Link className="country-card" key={index} to={`/${item.cioc}`}>
+            <img
+              src={item.flags.png}
+              alt={item.flags.alt}
+              className="country-image"
+            />
+            <div className="country-content">
+              <h3>
+                {item.name.common}
+                {' '}
+              </h3>
+              <p>
+                Population:
+                {' '}
+                <span>{item.population}</span>
+              </p>
+              <p className="more">
+                Click for more details
+                {' '}
+                <BsFillArrowRightCircleFill />
+              </p>
+            </div>
+          </Link>
+        ))
+      )}
+    </section>
```

Comment:
A good job has been done so far, but there are a few changes that would be great if you could consider tweaking in order for your project to match the given design. Kindly follow the points highlighted below:

- [ ] It would be nice if you could remove the placeholder text and keep communication simple as the original design does. For instance, for the Population: 1982189, you could remove the Population text and maintain the number as it's clearly communicated by the app as countries data and when the details button is clicked, more details can be found on the details page. 👍🏽
- [ ] Also, the text before the arrow indicating a link to the details page could be removed. You could remove the "Click for more details" text and then move the icon to the top just as it's in the original design and as marked in the screenshot below. 👍🏽
- [ ] Kindly for the home page, have the name of the countries in uppercase letters and then place both the name of the country and its population at the bottom right of the single country card. Please reference the original design in the screenshot below. 👍🏽

**THE ORIGINAL DESIGN**

<img width="411" alt="Screenshot 2023-04-18 at 10 56 34 AM" src="https://user-images.githubusercontent.com/50941074/232761926-270b4b06-24bb-40ff-913f-020e3991303c.png">


**YOUR DESIGN**

<img width="461" alt="Screenshot 2023-04-18 at 10 58 21 AM" src="https://user-images.githubusercontent.com/50941074/232761814-9dc14ead-4085-4d5d-aca4-2103a544fb21.png">




[ OPTIONAL CHANGE ]

- [ ] As React is built with component architecture in mind, you could extract this single component JSX code into a SingleCountry component and use it here by passing its data as props. That will make this component lean and clean and help utilize the main purpose of react, building smaller components as much as you can. 👍🏽

Reply:
Thank you!